docs/source/standard.md

```diff
@@ -4,27 +4,62 @@
 
 <!-- What this document intends to do (and what it doesn't), the level of detail we aim for contrasted with other technical standards -->
 
-The SATRE standard has been developed based on the following principles:
+## Design
 
-- TREs should be as as easy as possible for end-users to use (_e.g._ researchers) whilst still remaining secure.
-- TRE deployments should be offered that support data of different levels of sensitivity (_e.g._ through a tiered system of technical controls and policies).
-- TREs conforming to the standard should be interoperable and provide a familiar end-user experience.
-- The standard will be managed and updated following an open, community-driven process, and will not be tied to a single vendor or implementation.
+### Overview
+
+The SATRE specification has been designed to follow a capability-evaluation model.
+
+The specification is presented in terms of capabilities teams should aim for across all aspects of TRE provision.
+These capabilities in turn are broken down into components, which are represented by statements of processes, methods and practices teams should have in place to ensure they fulfil the capability requirements.
+
+Any particular component is labeled as `Mandatory`, `Recommended` or `Optional` in order to meet the specification.
+Currently, teams are able to either meet the specification (show they can fulfil all `Mandatory` components) or not.
+Future versions of the specification may introduce more granular levels of evaluation, for instance tiered level of accreditation based on fulfilment of `Mandatory`, `Recommended` and `Optional` components respectively.
 
-The SATRE standard is based around the idea of capabilities, some of which are required and some of which are optional.
 Any particular TRE implementation should be able to score itself against each capability as either "supported", "partially supported" or "unsupported" (see {ref}`evaluation` for details).
-The capabilities are grouped into the following broad categories:
 
-- {ref}`information governance <standard_capability_information_governance>`
-- {ref}`computing technology <standard_capability_computing_technology>`
-- {ref}`data management <standard_capability_data_management>`
-- {ref}`information security <standard_capability_information_security>`
+### Architecture
+
+```{figure} ../images/Architecture.svg
+:alt: SATRE Specification Architecture
+:align: center
+```
```

Comment:
Could make the components more generic eg. `Component 1` etc. rather than promising we'll have controls, business processes, applications and tech components for each capability